tron/utils/scribereader.py

```diff
@@ -59,15 +64,7 @@ def get_ecosystem() -> str:
 
 
 @lru_cache(maxsize=1)
-def get_scribereader_host_and_port() -> Optional[Tuple[str, int]]:
-    try:
-        ecosystem = get_ecosystem()
-        superregion = get_superregion()
-        region = get_region()
-    except OSError:
-        log.warning("Unable to read location mapping files from disk, not returning scribereader host/port")
-        return None
-
+def get_scribereader_host_and_port(ecosystem, superregion, region: str) -> Optional[Tuple[str, int]]:
     # NOTE: Passing in an ecosystem of prod is not supported by scribereader
```

Comment:
still valid, as in case of using scirbereader tailler it needs to get scribe service host and port, which will raise KeyError when called with "prod" ecosystem argument.